interfaces/seccomp/template.go

```diff
@@ -738,7 +738,7 @@ setresuid32 u:###USERNAME### u:###USERNAME### u:root
 setresuid u:###USERNAME### u:root u:root
 setresuid32 u:###USERNAME### u:root u:root
 
-# allow chown to ###USERNAME###:###GROUP###
+# allow chown to ###USERNAME###:###GROUP##
```

Comment:
this looks like a fat finger or something

Reply:
Indeed, silly me :( Fixed via force push